modules/library/referencing/src/main/java/org/geotools/referencing/datum/DefaultPrimeMeridian.java

```diff
@@ -210,4 +213,9 @@ protected String formatWKT(final Formatter formatter) {
         formatter.append(getGreenwichLongitude(context));
         return "PRIMEM";
     }
+
+    @Override
+    public String formatPROJ(final PROJFormatter formatter) {
+        return !formatter.isDatumProvided() && formatter.isPrimeMeridianProvided() ? "+pm=" : "";
```

Comment:
Seeing all these also makes me wonder if it wouldn't be better for the formatted to just collect a map of key/values, and generate the string and the end. Seems like it would require less status tracking.

Reply:
At the end we are only tracking 3 components datum, ellipsoid and primemeridian.
The formatting may involve nested objects which may add additional string definitions.
Formatting an IdentifiedObject/Formattable through the formatter will eventually return the keyword that the formatter should prepend when retrieving the name of that identifiedObject.